stdlib/public/Platform/android.modulemap

```diff
@@ -0,0 +1,661 @@
+  header "jni.h"
+  export *
+}
+
+module _langinfo [system] {
+  header "langinfo.h"
+  export *
+}
+
+module _pthread [system] {
+  header "pthread.h"
+  export *
+}
+
+module _regex [system] {
+  header "regex.h"
+  export *
+}
+
+module _resolv [system] {
+  header "resolv.h"
+  export *
+}
+
+module _sched [system] {
+  header "sched.h"
+  export *
+}
+
+module _search [system] {
+  header "search.h"
+  export *
+}
+
+module _semaphore [system] {
+  header "semaphore.h"
+  export *
+}
+
+module _spawn [system] {
+  header "spawn.h"
+  export *
+}
+
+module _strings [system] {
+  header "strings.h"
+  export *
+}
+
+module _sys [system] {
+  module syscall {
+    header "syscall.h"
+    export *
+  }
+  module sysexits {
+    header "sysexits.h"
+    export *
+  }
+  module syslog {
+    header "syslog.h"
+    export *
+  }
+  export *
+}
+
+module _tar [system] {
+  header "tar.h"
+  export *
+}
+
+module _termio [system] {
+  module termio {
+    header "termio.h"
+    export *
+  }
+  module termios {
+    header "termios.h"
+    export *
+  }
+  export *
+}
+
+module _threads [system] {
+  header "threads.h"
+  export *
+  explicit module threads_inlines {
+    header "bits/threads_inlines.h"
+    export *
+  }
+}
+
+module _uconfig_local [system] {
+  header "uconfig_local.h"
+  export *
+}
+
+module _ucontext [system] {
+  header "ucontext.h"
+  export *
+}
+
+module _wait [system] {
+  header "wait.h"
+  export *
+  explicit module sys_wait {
+    header "sys/wait.h"
+    export *
+  }
+}
+
+module _wchar_uchar [system] {
+  module uchar {
+    header "uchar.h"
+    export *
+  }
+  module wchar {
+    header "wchar.h"
+    export *
+  }
+  explicit module mbstate_t {
+    header "bits/mbstate_t.h"
+    export *
+  }
+  export *
+}
+
+module _xlocale [system] {
+  header "xlocale.h"
+  export *
+}
+
+// Additional modules in the 'android' subdirectory.
+module _android_defs [system] {
+  module ndk_version {
+    header "android/ndk-version.h"
+    export *
+  }
+  module versioning {
+    header "android/versioning.h"
+    export *
+  }
+  export *
+}
+
+module _android_apis [system] {
+  module asset_manager_jni {
+    header "android/asset_manager_jni.h"
+    export *
+  }
+  module asset_manager {
+    header "android/asset_manager.h"
+    export *
+  }
+  module log {
+    header "android/log.h"
+    export *
+  }
+  module trace {
+    header "android/trace.h"
+    export *
+  }
+  export *
+}
+
+// Additional modules in the 'bits' subdirectory.
+module _bits_io_inet [system] {
+  module in_addr {
+    header "bits/in_addr.h"
+    export *
+  }
+  module ioctl {
+    header "bits/ioctl.h"
+    export *
+  }
+  module ip_mreq_source {
+    header "bits/ip_mreq_source.h"
+    export *
+  }
+  module ip_msfilter {
+    header "bits/ip_msfilter.h"
+    export *
+  }
+  module sa_family_t {
+    header "bits/sa_family_t.h"
+    export *
+  }
+  export *
+}
+module _bits_signaltypes [system] {
+  header "bits/signal_types.h"
```

Comment:
Similarly these next ones should probably be submodules in signal, stdatomic, time